tests/sources/generate_diff.py

```diff
@@ -66,6 +67,7 @@ def github_api(request):
 filename = os.path.join(os.getenv('WEBOTS_HOME'), 'tests', 'sources', 'modified_files.txt')
 if j['total_count'] == 0:  # if no PR is associated with this commit, create an empty modified_files.txt to disable the tests
```

Comment:
This is clearly problematic as it will allow to create the branch push it, wait for the test to pass and then open the PR.

Reply:
See this example which should clearly have failed: #1591